### PR TITLE
Make Web Application Progressive

### PR DESCRIPTION
**Pull request by shohrab-uddin**
## Title
Title of the pattern

## Version
Designation of iteration on the pattern. This will initially be assigned by the patterns working group

## Submitted By
The name of the person(s) submitting the pattern

## Published Date
The date this version of the pattern is published. This will be provided by the patterns working group upon approval

## Intent
Subtitle describing what this pattern is expected to do

## Tags
Pre-defined list of tags which might apply to the pattern (e.g. Cloud, Web)
```
---
tags:
 - cloud
 - web
---
```

## Problem
What is the problem this pattern is solving

## Solution
How will this patter solve the problem

## SCI Impact
How will this pattern affect an SCI score of an application and why

```
`SCI = (E * I) + M per R`
```

## Assumptions
What are the assumptions being made

## Pros & Cons
Discussion section for pros and cons of this pattern
```
- **PRO**:
- **CON**:
```

### CODE REVIEW

**Comment by holanita** [comment]
@greenhsu123 pls review or assign to the right SME

**Comment by greenhsu123** [comment]
Hi @shohrab-uddin, thank you for your submission and PR. 

Unfortunately, the solution you have provided is too high level and does not qualify as a Pattern. I suggest reading our current [Web Patterns](https://github.com/Green-Software-Foundation/green-software-patterns/tree/main/docs/catalog/web). 

We are looking for how a specific intention will impact the carbon emission of a problem. For example, could you specifically mention a technique of Progressive Web Application (perhaps, caching static content?) that impacts the size of data transmitted?

Thanks again for your contribution, and please let me know if you need more information.

**Comment by shohrab-uddin** [comment]
Hello Sarah,

Thank you for your review. I will try to narrow it down and send another PR
soon.

Kind Regards
Shohrab Uddin

File: docs/catalog/ai/index.md [hunk 1]
```diff
@@ -11,5 +11,4 @@ import Image from "@theme/IdealImage";
 
 <TeamMembers>
     <TeamMember github="navveenb">Navveen Balani</TeamMember>
-    <TeamMember github="tmcclell">Tammy McClellan</TeamMember>
```
Comment (markus-ntt-seidl) [inline docs/catalog/ai/index.md:14]: I'm not sure what happened here, but PRs should only push patterns and not changes to the website structure. Can you please clean this PR?

**Comment by markus-ntt-seidl** [comment]
@shohrab-uddin 
Will you update this pattern in this PR or send a different PR? If you send a different PR we could close this one?

**Comment by shohrab-uddin** [comment]
Hi

You may close the PR. I will send a new one soon

Kind Regards
Shohrab

**Pull request closed without merging**